tests/test_snapshots/test_nft_snapshot.py

```diff
@@ -0,0 +1,67 @@
+from decimal import Decimal
+from typing import Dict
+from unittest import TestCase
+
+import pytest
+
+from helpers.constants import BADGER
+from helpers.enums import Network
+from rewards.snapshot.nft_snapshot import nft_snapshot_usd
+
+
+@pytest.fixture(autouse=True)
```

Comment:
Please, don't use autouse for fixtures that are not in the conftest.py. This function will be applied to ALL tests in the project before each test run and it will be confusing. 
Suggest to inject this function into each test where you need it

Reply:
Ok